src/azure-cli/service_name.json

```diff
@@ -11,8 +11,8 @@
   },
   {
     "Command": "az ad",
-    "AzureServiceName": "Azure Active Directory",
-    "URL": "https://learn.microsoft.com/azure/active-directory"
+    "AzureServiceName": "Microsoft Entra ID (Azure Active Directory)",
```

Comment:
https://learn.microsoft.com/en-us/entra/identity/ still contains the old name:

> Microsoft Azure Active Directory (Azure AD) is being renamed to Microsoft Entra ID. 

Shall we do the same?

Reply:
Because our reference group name is not changing, I feel we need to say "formerly Azure Active Directory". or "formerly named" or "formerly known as".  Just putting AAD in `()` could be misinterpreted that AAD is still an active name.

Reply:
Surely I can do this:

```suggestion
    "AzureServiceName": "Microsoft Entra ID (formerly known as Azure Active Directory)",
```

The only concern is that this text will appear in the table of contents. Won't it be too long to fit in there?

![image](https://github.com/Azure/azure-cli/assets/4003950/cf2d34ed-fbe7-4562-b501-d564d7be0906)

Reply:
I will use `Microsoft Entra ID (formerly known as Azure Active Directory)`.